sdk/python/tests/compiler/testdata/withitem_basic.py

```diff
@@ -57,3 +57,16 @@ def pipeline(my_pipe_param=10):
 if __name__ == '__main__':
     from kfp import compiler
     print(compiler.Compiler().compile(pipeline, package_path=None))
+
+    import kfp
+    client = kfp.Client(host='127.0.0.1:8080/pipeline')
```

Comment:
Are you sure this code should be part of the sample?

Reply:
it isn't run as part of the test, but it's useful if you want to easily one-off manually test the test.